hack/build-jsonnet.sh

```diff
@@ -37,7 +37,7 @@ for file in "${files[@]}"; do
 	}&
 
 	# wait for at least one of the jobs to finish if there are more than maxProc jobs
-	while [[ $(jobs -r | wc -l ) -ge "$maxProc" ]]; do wait -n; done
+	while [[ $(jobs -r | wc -l ) -ge "$maxProc" ]]; do wait; done
```

Comment:
I think we can revert this and address it clearly in another PR.
It seems your installed `wait` doesn't support `-n`, I'm afraid remove it would break sth for the others.

Reply:
Ya that was not intentional, I reverted it